src/App.vue

```diff
@@ -55,6 +55,26 @@ onMounted(() => {
   window
     .matchMedia('(prefers-color-scheme: dark)')
     .addEventListener('change', setTheme);
+
+  const handleKeyDown = (e) => {
+      const key = e.key;
+      let keyDetected = true;
+
```

Comment:
this code needs a guard to handle the following cases:
- api connection not (yet) initialized or lost
- no active player 
- no queue actions available